sdk/iOS/OoyalaSkinSDK/React/OOReactSkinEventsEmitter.m

```diff
@@ -0,0 +1,105 @@
+/**
+ * @file       OOReactSkinEventsEmitter.m
+ * @class      OOReactSkinEventsEmitter OOReactSkinEventsEmitter.m "OOReactSkinEventsEmitter.m"
+ * @brief      OOReactSkinEventsEmitter
+ * @details    OOReactSkinEventsEmitter.m in OoyalaSDK
+ * @date       13/8/18
```

Comment:
This is just nitpicking but can you write it as 8/13/18. Then it'll be consistent with the rest of the code.